test/jasmine/tests/splom_test.js

```diff
@@ -549,6 +549,21 @@ describe('@gl Test splom interactions:', function() {
         .catch(failTest)
         .then(done);
     });
+
+    it('should work with typed arrays', function(done) {
```

Comment:
locking a fix for https://github.com/plotly/plotly.js/issues/2585 from the new `regl-splom` version